src/install.rs

```diff
@@ -254,6 +259,31 @@ fn write_firstboot_kargs(mountpoint: &Path, args: &str) -> Result<()> {
     Ok(())
 }
 
+/// Write persistent kernel arguments.
+fn write_persistent_kargs(mountpoint: &Path, args: &str) -> Result<()> {
+    eprintln!("Writing persistent kernel arguments");
+
+    edit_bls_entries(mountpoint, |orig_contents: String| {
```

Comment:
For this and the other similar closure below, what about moving it out of this scope instead? I'd be really happier to see both covered with a few positive and negative unit tests.

Reply:
This is part of https://github.com/coreos/coreos-installer/pull/271.